reporters/default_reporter.go

```diff
@@ -149,10 +149,8 @@ func (r *DefaultReporter) DidRun(report types.SpecReport) {
 			}
 		} else {
 			header, stream = denoter, true
-			if report.NumAttempts > 1 {
-				if report.IsFlakeAttempts {
-					header, stream = fmt.Sprintf("%s [FLAKEY TEST - TOOK %d ATTEMPTS TO PASS]", r.retryDenoter, report.NumAttempts), false
-				}
+			if report.MaxFlakeAttempts > 1 {
```

Comment:
I don't think this is right.  You need `report.NumAttempts >  1 && report.MaxFlakeAttempts > 1` otherwise every spec marked with `MaxFlakeAttempts(N)` will get this output.

The intent is to only tell the user "hey this spec was flakey" if it actually takes more than one try to succeed.  Otherwise they should just see a regular passing spec.